runtime/vm/jfr.cpp

```diff
@@ -729,6 +748,90 @@ jfrExecutionSampleCallback(J9VMThread *currentThread, IDATA handlerKey, void *us
 	jfrExecutionSample(currentThread, currentThread);
 }
 
+void
+jfrCPULoad(J9VMThread *currentThread)
+{
+	PORT_ACCESS_FROM_VMC(currentThread);
+	OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
+
+	omrthread_process_time_t currentProcCPUTime = {0};
+	intptr_t processTimeRC = omrthread_get_process_times(&currentProcCPUTime);
+
+	J9SysinfoCPUTime currentSysCPUTime = {0};
+	intptr_t sysTimeRC = omrsysinfo_get_CPU_utilization(&currentSysCPUTime);
+
+	if (0 == processTimeRC && 0 == sysTimeRC) {
+		J9JFRCPULoad *jfrEvent = (J9JFRCPULoad *)reserveBuffer(currentThread, sizeof(J9JFRCPULoad));
+		if (NULL != jfrEvent) {
+			initializeEventFields(currentThread, (J9JFREvent *)jfrEvent, J9JFR_EVENT_TYPE_CPU_LOAD);
+			UDATA numberOfCpus = j9sysinfo_get_number_CPUs_by_type(J9PORT_CPU_ONLINE);
+			U_64 currentTime = (U_64)j9time_nano_time();
```

Comment:
This should not mix types: it should use `int64_t` as declared by OMR. Related locals should use `int64_t` instead of `I_64`.

Reply:
Do you suggest changing `prevProcTimestamp` fields to `int64_t` as well?

Reply:
Yes, they should use the same type as the source (`int64_t omrtime_nano_time()`).